VAEES/vaees.github.io
# Feature/hana onboarding

# Overview 
This branch contains the HANA onboarding data.

## Test Script 
1. Run docker-compose up
2. Open [HANA onboarding](http://localhost:4000/2021/03/01/ui5-onboarding/)
3. Check if the content is nice for new VAEES players

## Test Data 
N/A
